src/Scaffold/Console/model/settings-model.stub

```diff
@@ -0,0 +1,18 @@
+<?php namespace {{studly_author}}\{{studly_plugin}}\Models;
+
+use Model;
+
+/**
+ * {{name}} Model
+ */
+class {{studly_name}} extends Model
+{
+    public $table = '{{lower_author}}_{{lower_plugin}}_{{snake_plural_name}}';
+    use \Winter\Storm\Database\Traits\Validation;
+
+    public $implement = ['System.Behaviors.SettingsModel'];
+
+    public $settingsCode = '{{lower_author}}_{{lower_plugin}}_{{lower_name}}';
+
+    public $settingsFields = 'fields.yaml';
+}
```

Comment:
Would be nice to generate the model with comments. Also the table property doesn't need to be defined for settings models but we do need to define rules if the validation trait is in use.

```suggestion
<?php namespace {{studly_author}}\{{studly_plugin}}\Models;

use Model;

/**
 * {{name}} Model
 */
class {{studly_name}} extends Model
{
    use \Winter\Storm\Database\Traits\Validation;

    /**
     * @var array Behaviors implemented by this model.
     */
    public $implement = ['System.Behaviors.SettingsModel'];

    /**
     * @var string Unique code
     */
    public $settingsCode = '{{lower_author}}_{{lower_plugin}}_{{lower_name}}';

    /**
     * @var mixed Settings form field definitions
     */
    public $settingsFields = 'fields.yaml';

    /**
     * @var array Validation rules
     */
    public $rules = [];
}

```